tests/fixtures/semantic_parsing/nlvr_coverage_semantic_parser/serialization/vocabulary/denotations.txt

```diff
@@ -0,0 +1,2 @@
+false
```

Comment:
Did you mean to commit an untar-ed `model.tar.gz`?

Reply:
This is what gets generated when you run `train_fixtures.py`.  I don't think it's a problem to have this here; seeing the vocabulary files from the test fixtures can actually be helpful.

Reply:
Good to know. I agree that this is useful to look at, just wasn't sure if this was intended to be checked it.